docs/guides/platform/Upgrade_IBM_i_Modernization_Engine_for_Lifecycle_Integration.md

```diff
@@ -1,29 +1,57 @@
-    Based on the OLM Subscription, the IBM i Developer Tool operator is upgraded either automatically or after manual approval.
-        This will roll out an update to the operator-controller-manager pod, but the IBM i Developer Customer Resource component will not be changed.
-    The namespace administrator chooses an available version and updates the spec.version field on the IBM i Developer Customer Resource custom resource.
-    The change to the desired version triggers reconciliation of the IBM i Developer Customer Resource resource and upgrades the IBM i Developer Tool components.
+If you have the Merlin operator installed in single namespace mode as in case 2, you will need to first patch the operator group before changing the subscription channel.
 
-Rollback
+## Patching the operator group to use All Namespaces Mode (Only if you were in case 2)
+1. Login to your openshift cluster using the openshift cli. `oc login -u <username> -p <password> <openshift-server-url>`
+2. Change to use the project that your Merlin instance is installed in. `oc project <Merlin-project>`
+3. Get the operator group name. `oc get operatorgroup`
+4. Patch the operator group using the name you found in the previous step. `oc patch operatorgroup <operatorgroup-name> --type='json' -p='[{"op": "remove", "path": "/spec/targetNamespaces"}]'`
 
-The versions of the operator and operand are decoupled. A given version of the operator may support multiple versions of the operand. If an issue is encountered after an upgrade, the previous stable state can be restored by changing the IBM i Developer Customer Resource resource’s spec.version back to the previous value. OLM does not support reverting the version of the installed operator, but this should not be necessary to restore a stable state to the IBM i Developer Tool components.
-Procedure
+## Changing the subscription channel
+**The next step is to change the subscription channel.**
+1. Login to the openshift web console, and go the project that your Merlin instance is installed in.
+2. **(Only do this step if you had to patch the operatorgroup. Otherwise, skip to step 3)**
+Wait until ibm-cloudpak operator shows an error about intersecting operator groups. This is how you will know the operator group patch completed successfully. Then delete the ibm-cloudpak operator instance and operator in current project.
+3. Change the subscription channel to v2, and wait for the install to complete.
+![Patch operator subscription](../../images/upgrade/patchOperatorSubscription.png)
+4. If you installed the Merlin operator using the manual upgrade strategy, [follow these steps to manually upgrade the operator](./guides/platform/upgrade_merlin_operator).
+
+## Upgrading the installed applications
+There are two possible cases you can be in when you are upgrading the applications.
+
+Case 1: You have not installed the IDE and CICD applications in the same project\
```

Comment:
The names are IBM i Developer Tool and IBM i CI/CD Tool (this applies to other instances in the document)

Reply:
Done